src/panoptes/pocs/camera/camera.py

```diff
@@ -559,19 +562,21 @@ def process_exposure(self,
         Raises:
             FileNotFoundError: If the FITS file isn't at the specified location.
         """
+        # Wait for exposure to complete. Timeout handled by exposure thread.
+        while self.is_exposing:
+            time.sleep(1)
+
+        self.logger.debug(f'Starting exposure processing for {observation_event}')
+
         # If passed an Event that signals the end of the exposure wait for it to be set
         compress_fits = compress_fits or self.get_config('observations.compress_fits')
         make_pretty_images = make_pretty_images or self.get_config('observations.make_pretty_images')
 
-        # Wait for exposure to complete.
-        while self.is_exposing:
-            time.sleep(1)
-
-        image_id = info['image_id']
-        seq_id = info['sequence_id']
-        file_path = info['file_path']
-        exptime = info['exptime']
-        field_name = info['field_name']
+        image_id = metadata['image_id']
```

Comment:
One way to clean this up a bit (and make it more flexible) would be to offload these mappings to a `header_mappings.yaml` file. Same goes for `_process_exposure`.

Reply:
Yeah, I was just being lazy with this PR and didn't want it to grow too much. Will make issue.